### ISSUE TITLE
Command line help contains instructions for --public parameter that doesn't actually exist

### ISSUE THREAD

**Issue by lumaxis** [gorzell/folding-at-cloud#1]
The output of `start.sh -h` contains a `-p | --public` parameter that does not actually exist or work

**Comment by gorzell** [comment]
Ya, I was hoping to provide away to not expose the host to the internet unless people actually wanted that. I just didn't have a chance to implement that yet and didn't want it to be a blocker.

**Comment by lumaxis** [comment]
Makes sense 🙂 Wasn't really a problem, just found it confusing and wanted to write it down somewhere.
